README.md

```diff
@@ -2,207 +2,219 @@
-    console.log('selected day', day);
-  }}
-  // Month format in calendar title. Formatting values: http://arshaw.com/xdate/#Formatting
-  monthFormat={'yyyy MM'}
-  // Handler which gets executed when visible month changes in calendar. Default = undefined
-  onMonthChange={month => {
-    console.log('month changed', month);
-  }}
-  // Hide month navigation arrows. Default = false
-  hideArrows={true}
-  // Replace default arrows with custom ones (direction can be 'left' or 'right')
-  renderArrow={direction => <Arrow />}
-  // Do not show days of other months in month page. Default = false
-  hideExtraDays={true}
-  // If hideArrows = false and hideExtraDays = false do not switch month when tapping on greyed out
-  // day from another month that is visible in calendar page. Default = false
-  disableMonthChange={true}
-  // If firstDay=1 week starts from Monday. Note that dayNames and dayNamesShort should still start from Sunday
-  firstDay={1}
-  // Hide day names. Default = false
-  hideDayNames={true}
-  // Show week numbers to the left. Default = false
-  showWeekNumbers={true}
-  // Handler which gets executed when press arrow icon left. It receive a callback can go back month
-  onPressArrowLeft={subtractMonth => subtractMonth()}
-  // Handler which gets executed when press arrow icon right. It receive a callback can go next month
-  onPressArrowRight={addMonth => addMonth()}
-  // Disable left arrow. Default = false
-  disableArrowLeft={true}
-  // Disable right arrow. Default = false
-  disableArrowRight={true}
-  // Disable all touch events for disabled days. can be override with disableTouchEvent in markedDates
-  disableAllTouchEventsForDisabledDays={true}
-  // Replace default month and year title with custom one. the function receive a date as parameter
-  renderHeader={date => {
-    /*Return JSX*/
-  }}
-  // Enable the option to swipe between months. Default = false
-  enableSwipeMonths={true}
-/>
+import React, { useState } from 'react';
+import { Calendar, LocaleConfig } from 'react-native-calendars';
+
+const App = () => {
+  const [selected, setSelected] = useState('');
+
+  return (
+    <Calendar
+      style={{
+        borderWidth: 1,
+        borderColor: 'gray',
+        height: 350,
+      }}
+      theme={{
+        backgroundColor: '#ffffff',
+        calendarBackground: '#ffffff',
+        textSectionTitleColor: '#b6c1cd',
+        selectedDayBackgroundColor: '#00adf5',
+        selectedDayTextColor: '#ffffff',
+        todayTextColor: '#00adf5',
+        dayTextColor: '#2d4150',
+        textDisabledColor: '#d9e
 ```
+## Customized Calendar Examples
 
-#### Date marking
+### Calendar
+<kbd>
+  <img src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/calendar.gif?raw=true">
+</kbd>
 
-**Disclaimer**: Make sure that `markedDates` param is immutable. If you change `markedDates` object content but the reference to it does not change calendar update will not be triggered.
```

Comment:
Is this note not important? Can we maybe add a info box with all the important notes along this file?